### PR TITLE
README.md: fixed other language table

### PR DESCRIPTION
**Pull request by shashankx86**
fixed other language table:
- added language in there language

### CODE REVIEW

**Comment by ascodeasice** [comment]
@TenTraicion Should we add translation of each language in the language table?
Also, the field in `Chinese (Traditional)` should be "繁體中文" (traditional chinese), instead of "中國人"(chiense people)

**Comment by shashankx86** [comment]
@ascodeasice Fixed

**Comment by TenTraicion** [comment]
@ascodeasice, we only need to list every single translation in `Translations/README.md` and the main files on the root directory.
the translated docs aren't mandatory to have language table. but a link to `Translations/README.md` is mandatory though.

make sure the table is ordered alphabetically.

**Comment by shashankx86** [comment]
> make sure the table is ordered alphabetically.
@TenTraicion they are

File: README.md [hunk 1]
```diff
@@ -82,11 +82,11 @@ You will make a copy of the card template inside the HTML file and customize it
...
-|  [German](/translations/README/GERMAN.md)  |      [Hindi](/translations/README/HINDI.md)      | [Italian](/translations/README/ITALIAN.md) | [Japanese](/translations/README/JAPANESE.md) |  [Korean](/translations/README/KOREAN.md)  |
-  [Polish](/translations/README/POLISH.md)  | [Portuguese](/translations/README/PORTUGUESE.md) | [Russian](/translations/README/RUSSIAN.md) |  [Serbian](/translations/README/SERBIAN.md)  | [Spanish](/translations/README/SPANISH.md) |
-   [Turkish](/translations/README/TURKISH.md) | [Ukrainian](/translations/README/UKRAINIAN.md) |
+|  [German (Deutsch)](/translations/README/GERMAN.md)  |      [Hindi (हिंदी)](/translations/README/HINDI.md)      | [Italian (Italiano)](/translations/README/ITALIAN.md) | [Japanese (日本語)](/translations/README/JAPANESE.md) |  [Korean (한국어)](/translations/README/KOREAN.md)  |
+  [Polish (Polski)](/translations/README/POLISH.md)  | [Portuguese (Portuguese)](/translations/README/PORTUGUESE.md) | [Russian (Русский)](/translations/README/RUSSIAN.md) |  [Serbian (Српски)](/translations/README/SERBIAN.md)  | [Spanish](/translations/README/SPANISH.md) |
```
Comment (ascodeasice) [inline README.md:88]: Spanish is missing, add this and I'll merge this commit. Thanks for your contribution

```suggestion
  [Polish (Polski)](/translations/README/POLISH.md)  | [Portuguese (Portuguese)](/translations/README/PORTUGUESE.md) | [Russian (Русский)](/translations/README/RUSSIAN.md) |  [Serbian (Српски)](/translations/README/SERBIAN.md)  | [Spanish (Español)](/translations/README/SPANISH.md) |
```

**Comment by ascodeasice** [comment]
Thanks for your contribution, you can also do the same thing with [`/translations/README.md`](https://github.com/Syknapse/Contribute-To-This-Project/blob/master/translations/README.md), which would be helpful.

**Pull request merged**